src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -0,0 +1,476 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import org.apache.spark.serializer.KryoRegistrator;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Various types of read anomalies that provide evidence of genomic breakpoints.
```

Comment:
I think I'm understanding this more and more as I read through the code, but it would be helpful to expand this comment to give a little more information about the data held by this object.. Ie. each instance of BreakpointEvidence essentially maps to 1 read and/or read pair, etc.